### PR TITLE
feat: decimalPlaces for wellKnownBrands

### PR DESCRIPTION
**Pull request by turadg**

### CODE REVIEW

File: src/utils/displayFunctions.ts [hunk 1]
```diff
@@ -23,10 +24,10 @@ export const makeDisplayFunctions = (brandToInfo: Map<Brand, BrandInfo>) => {
   const getDecimalPlaces = (brand: Brand) => {
     // XXX for rpc brands that don't come in the purse watcher
     if ('boardId' in brand) {
-      switch ((brand as any).boardId) {
-        case 'board0188':
-          return 6;
-      }
+      const { boardId } = brand as unknown as { boardId: string };
+      const info = wellKnownBrands[boardId];
+      assert(info, `unknown boardId ${boardId}`);
```
Comment (dckc) [inline src/utils/displayFunctions.ts:29]: crash the app in this case? I guess that's good enough for now...

**Review by dckc — APPROVED** [review]
this should work.

File: src/config.ts [hunk 2]
```diff
@@ -3,3 +3,12 @@ export const dappConfig = {
   INSTANCE_PREFIX: ':published.psm.IST.',
   INSTANCES_KEY: ':published.agoricNames.instance',
 };
+
+export const wellKnownBrands = {
```
Comment (dckc) [inline src/config.ts:7]: we could also derive that other list of tokens from this one.

Comment (turadg) [reply src/config.ts:7]: yeah, though it's different for mainnet and tests, right? a different set

File: src/utils/displayFunctions.ts (hunk 1 above)
Comment (turadg) [reply src/utils/displayFunctions.ts:29]: I figure, given how quickly we can fix things, better to error on noisy so we know about problems

**Pull request merged**